src/vsphere_cpi/lib/cloud/vsphere/vm_creator.rb

```diff
@@ -261,7 +262,6 @@ def create(vm_config)
             end
             unless vm_config.pci_passthroughs.empty?
               config_spec = VimSdk::Vim::Vm::ConfigSpec.new
-              config_spec.device_change = []
```

Comment:
Is there any concern with this no longer being set to anything in some cases? Previously this was set to empty array, but now it only gets configured to something if root_disk_size_gb > 0.

If you don't specify root_disk_size_gb, this won't get set to empty array. Is that okay?

Reply:
> Is there any concern with this no longer being set to anything in some cases?

Actually, it's *always* set to an empty array. When you create a new `VimSdk::Vim::Vm::ConfigSpec` (which is `config_spec`'s class), it initializes the `device_change` accessor to an empty array. There's no reason to set it to an empty array a second time.